com.abisource.AbiWord.json

```diff
@@ -0,0 +1,218 @@
+{
+    "app-id" : "com.abisource.AbiWord",
+    "branch" : "3.0",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "abiword",
+    "tags" : [
+       "stable"
+    ],
+    "finish-args" : [
+        "--allow=devel",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
```

Comment:
`xdg-document` would be much better, or making use of xdg-desktop-portal, especially if it's gtk3.

Reply:
I need to actually check AbiWord works with portals.

Reply:
The major thing to look at as a GTK app is if you use FileChooserNative (you should)

Reply:
AbiWord doesn't as the requiremnt for Gtk+ is 3.12 in the stable branch. I'm ok bumpimg to 3.20 for the unstable branch though.

Reply:
I use --filesystem=home for now